client/components/editor/teaching-elements/Assessment/feedback/FeedbackItem.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+  <li>
+    <span class="answer-index">Answer {{ index + 1 }}:</span>
+    <template v-if="answer.length">{{ answer }}</template>
+    <i v-else>Answer not added.</i>
+    <quill-editor
+      v-if="isEditing"
+      :content="feedback"
+      :options="quillOptions"
+      @change="$emit('update', $event)">
+    </quill-editor>
+    <div v-else-if="feedback.length" v-html="feedback"></div>
```

Comment:
Isn't this essentially:
```html
<template v-if="feedback.length">
  <quill-editor
    v-if="isEditing"
    :content="feedback"
    :options="quillOptions"
    @change="$emit('update', $event)">
  </quill-editor>
  <div v-else v-html="feedback"></div>
</template>
```
which is kinda _flattened_ but that makes it harder to reason about? 🤔